src/storage/src/statistics.rs

```diff
@@ -235,3 +320,51 @@ impl SourceStatistics {
         cur.2.bytes_received.inc_by(value);
     }
 }
+
+impl StorageStatistics<SinkStatisticsUpdate, SinkStatisticsMetrics> {
+    pub(crate) fn new(id: GlobalId, worker_id: usize, metrics: &SinkBaseMetrics) -> Self {
+        Self {
+            stats: Rc::new(RefCell::new((
+                // We have no snapshot metrics for sinks as of now.
+                true,
+                SinkStatisticsUpdate {
+                    id,
+                    worker_id,
+                    messages_staged: 0,
```

Comment:
very much a nit: ordering is still different :sweat_smile:

Reply:
the _one_ place I missed!